services/movie_session.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+from typing import List
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    movie = Movie.objects.get(pk=movie_id)
+    cinema_hall = CinemaHall.objects.get(pk=cinema_hall_id)
+
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall=cinema_hall,
+        movie=movie,
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> List[MovieSession]:

```

Comment:
You return not a list, but rather a `QuerySet` of `MovieSession`. Change your type annotation to encounter that.